config/initializers/mimemagic_office_files.rb

```diff
@@ -0,0 +1,11 @@
+# Add Office 2007+ file types to MimeMagic
```

Comment:
Is this something we'll need to keep an eye on/keep up to date?

Reply:
Hopefully not - this data has been static in the gem (before being removed) for a while.

If anything, it might eventually find its way into the MIME database on Alpine and we can get rid of it entirely 🤞